### PR TITLE
Architectural Refactoring and Recategorization

### PR DESCRIPTION
**Pull request by SebastianWagner2**
Contributes to: #65

Created class RemoteRepository as parent of Travis/GHARepo
Recategorization to eliminate different categories with same meaning (initialization vs. initialize-release vs. class initialize or action vs. behaviour vs. behavior, ...)

### CODE REVIEW

File: packages/AutoTDD-Core.package/ATDDGitHubActionsRepository.class/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+An ATDDGitHubActionsRepository facilitates the GHAClient and GHAParser to provide structured data.
```
Comment (benjamin-frost) [inline packages/AutoTDD-Core.package/ATDDGitHubActionsRepository.class/README.md]: Please write `GHAClient` and `GHAParser` out

File: packages/AutoTDD-Core.package/ATDDRemoteRepository.class/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+An ATDDRemoteRepository facilitates the Client and Parser to provide structured data.
```
Comment (benjamin-frost) [inline packages/AutoTDD-Core.package/ATDDRemoteRepository.class/README.md]: nit

```suggestion
An ATDDRemoteRepository facilitates a Client and Parser to provide structured data.
```

File: packages/AutoTDD-Core.package/ATDDRemoteRepository.class/instance/repository..st [hunk 3]
```diff
@@ -1,6 +1,6 @@
-accessing
-repository: aString
-	
+initialization
+repository: aString
```
Comment (benjamin-frost) [inline packages/AutoTDD-Core.package/ATDDRemoteRepository.class/instance/repository..st]:
```suggestion
repository: aString
```

File: packages/AutoTDD-Core.package/ATDDRemoteRepository.class/instance/repositoryJson.st [hunk 4]
```diff
@@ -1,4 +1,4 @@
-accessing
+initialization
```
Comment (benjamin-frost) [inline packages/AutoTDD-Core.package/ATDDRemoteRepository.class/instance/repositoryJson.st]: Why is this `initialization`?

File: packages/AutoTDD-Core.package/ATDDRemoteRepository.class/instance/update.st [hunk 5]
```diff
@@ -0,0 +1,4 @@
+initialization
```
Comment (benjamin-frost) [inline packages/AutoTDD-Core.package/ATDDRemoteRepository.class/instance/update.st]: Why is this `initialization`?

File: packages/AutoTDD-Core.package/ATDDRemoteRepository.class/instance/validate..st [hunk 6]
```diff
@@ -1,4 +1,4 @@
-data-processing
+initialization
```
Comment (benjamin-frost) [inline packages/AutoTDD-Core.package/ATDDRemoteRepository.class/instance/validate..st]: Why is this `initialization`?

File: packages/AutoTDD-GUI.package/ATDDInfo.class/class/testingEntityDictionary.st [hunk 7]
```diff
@@ -1,4 +1,4 @@
-instance creation
+initialization
```
Comment (benjamin-frost) [inline packages/AutoTDD-GUI.package/ATDDInfo.class/class/testingEntityDictionary.st]: Why is this `initialization`?

File: packages/AutoTDD-GUI.package/ATDDOverview.class/class/labelFor.and.and..st [hunk 8]
```diff
@@ -1,4 +1,4 @@
-instance creation
+accessing
```
Comment (benjamin-frost) [inline packages/AutoTDD-GUI.package/ATDDOverview.class/class/labelFor.and.and..st]: Does not sound like `accessing` to me

File: packages/AutoTDD-GUI.package/ATDDInfo.class/class/label.st [hunk 9]
```diff
@@ -1,4 +1,4 @@
-instance creation
+accessing
```
Comment (kolioOtSofia) [inline packages/AutoTDD-GUI.package/ATDDInfo.class/class/label.st]: not sure if this is really accessing

File: packages/AutoTDD-GUI.package/ATDDInfo.class/class/testingEntityDictionary.st (hunk 7 above)
Comment (kolioOtSofia) [inline packages/AutoTDD-GUI.package/ATDDInfo.class/class/testingEntityDictionary.st]: not initialization

File: packages/AutoTDD-GUI.package/ATDDWatcherEditor.class/instance/findSelectedItemsForUpdatedCollection.withSelected..st [hunk 10]
```diff
@@ -1,4 +1,4 @@
-utilities
+utility
 findSelectedItemsForUpdatedCollection: aCollection withSelected: selected
```
Comment (benjamin-frost) [inline packages/AutoTDD-GUI.package/ATDDWatcherEditor.class/instance/findSelectedItemsForUpdatedCollection.withSelected..st]: `selected` is probably not a great parameter name

File: packages/AutoTDD-Tests.package/ATDDBrowserTests.class/class/resetTestHasRun.st [hunk 11]
```diff
@@ -1,4 +1,4 @@
-as yet unclassified
+accessing
```
Comment (benjamin-frost) [inline packages/AutoTDD-Tests.package/ATDDBrowserTests.class/class/resetTestHasRun.st]: sounds more like an `action` to me

File: packages/AutoTDD-Tests.package/ATDDBrowserTests.class/class/setTestHasRun.st [hunk 12]
```diff
@@ -1,4 +1,4 @@
-as yet unclassified
+accessing
 setTestHasRun
```
Comment (benjamin-frost) [inline packages/AutoTDD-Tests.package/ATDDBrowserTests.class/class/setTestHasRun.st]: Should probably be renamed. No `set` and `get` in Squeak

**Review by benjamin-frost — CHANGES_REQUESTED** [review]

**Review by kolioOtSofia — CHANGES_REQUESTED** [review]

File: packages/AutoTDD-Tests.package/ATDDBrowserTests.class/class/setTestHasRun.st (hunk 12 above)
Comment (SebastianWagner2) [reply packages/AutoTDD-Tests.package/ATDDBrowserTests.class/class/setTestHasRun.st]: not a setter (takes no arguments): set and reset

**Review by benjamin-frost — APPROVED** [review]

**Review by kolioOtSofia — APPROVED** [review]

**Pull request merged**